lib/lrama/grammar.rb

```diff
@@ -446,6 +452,19 @@ def normalize_rules
       @rules.sort_by!(&:id).freeze
     end
 
+    # Add $accept rule to the top of rules
+    def add_accept_rules
```

Comment:
This method adds single accept rule then it's better to use `add_accept_rule` (without "s") or something like that.

Reply:
Change in https://github.com/ruby/lrama/pull/576/commits/4337cce0087f6d4d1562bbc94343ddf7f2927210